yoggi.py

```diff
@@ -136,8 +136,30 @@ def POST(self, request, response):
 
             mimetype = from_buffer(file.stream.read(1024), mime=True)
 
-            s3.put(path, file, response.user, mimetype)
+            public = response.params.get('public') or "False"
```

Comment:
I guess this is fine, but I think doing a database migration could be nice.

Reply:
Is that relevant here? Isn't this for uploading new files with a default public setting of `"False"`?

Reply:
@oskstr is right, if you don't provide the query parameter `?public=True` or `?public=False` it defaults to `false`